recruitments.html

```diff
@@ -208,276 +222,250 @@ <h2 class="black text-center">
 					Sensing And Automation
 				</h2>
 				<p>
-					The challenges for Sensing and Automation recruitments give you the opportunity to skip the interviews and get into the task
+					The challenges for Sensing and Automation recruitments give you the opportunity to skip the
+					interviews and get into the task
 					phase. Solve the questions below to grab the opportunity.
 					Deadline is 10th September.
 				</p>
```

Comment:
Change to the current deadline, Also add the date to other two divisions (Mech and management)